src/api/v1/services/dataset.service.ts

```diff
@@ -1,53 +1,91 @@
-import { Datasets } from '../models/dataset.model';
-
 import { dataset } from '../types/dataset.interface';
+import fs from 'fs';
 
 export default class DatasetService {
     public async getDatasets(q: string, offset: number, limit: number): Promise<dataset[]> {
-        const pipeline: Array<any> = [
+        // The files directory contains a set of example JSON files
+        const files = fs.readdirSync("./files");
```

Comment:
I don't know if you need to use a predefined file vs fetching from the db

Reply:
I asked Tristan to make static files, so we don't have to rely on potentially dodgy data from mk1, @dannitahdruk .